### ISSUE TITLE
2022-1-31

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#727]
501	TRANSFORMS; ToTensor(); Lambda Transforms; (2022-1-22)
502	Build the neural network; gpu’s name is cuda; 1, Get Device for Training; 2, Define the Class; 3, Model Layers; 4, Model parameters; (2022-1-23)
503	AUTOGRAD; 1, Tensors, Functions and Computational graph; 2, Computing Gradients; 3, Disabling Gradient Tracking; (2022-1-23)
504	Optimization; 1, Prerequisite Code; 2, Hyperparameters; 4, Full Implementation; (2022-1-23)
505	// Compute Unified Device Architecture，统一计算架构
506	Your CPU does computation on data in your computer’s RAM. Your GPU has dedicated memory attached to it. (2022-1-25)
507	Replace X with input; done; replace y with target; done; (2022-1-25)
508	Change device in optimize.py to gpu; done; (2022-1-25)
509	Change device in train_model.py to gpu; done; (2022-1-25)
